drivers/CAN.cpp

```diff
@@ -115,9 +115,11 @@ int CAN::filter(unsigned int id, unsigned int mask, CANFormat format, int handle
 void CAN::attach(Callback<void()> func, IrqType type) {
```

Comment:
Attach could also be called to replace a callback of the same type. If this is the case then deep sleep shouldn't be locked a second time.